postgres/README.md

```diff
@@ -189,7 +189,9 @@ See the PostgreSQL documentation on [`pg_hba.conf`](https://www.postgresql.org/d
 
 ### `PGDATA`
 
-This optional variable can be used to define another location - like a subdirectory - for the database files. The default is `/var/lib/postgresql/data`. If the data volume you're using is a filesystem mountpoint (like with GCE persistent disks) or remote folder that cannot be chowned to the `postgres` user (like some NFS mounts), Postgres `initdb` recommends a subdirectory be created to contain the data.
+> :warning: when mounting a volume to `/var/lib/posgresql`, the `/var/lib/postgresql/data` path is a local volume from the container runtime, thus data is not persistet on the mounted volume.
```

Comment:
Unfortunately, I don't think Docker Hub supports GitHub's emoji, so this will need to be an embedded literal emoji: (also a minor typo)

```suggestion
> ⚠ when mounting a volume to `/var/lib/posgresql`, the `/var/lib/postgresql/data` path is a local volume from the container runtime, thus data is not persisted on the mounted volume.
```

Additionally, `README.md` is generated (per the comment at the top of the file :sweat_smile:), so this change will need to be applied in `content.md` instead.

(I'm happy to assist from here if you'd prefer.)

Reply:
This is fixed now @tianon! Didn't. see the `contents.md` file 👍